pkg/kubelet/dockertools/manager.go

```diff
@@ -1873,7 +1873,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 		// Call the networking plugin
 		err = dm.networkPlugin.SetUpPod(pod.Namespace, pod.Name, podInfraContainerID)
 		if err != nil {
-			glog.Errorf("Failed to create pod infra container: %v; Skipping pod %q", err, podFullName)
+			glog.Errorf("Failed to setup pod networkplugin: %v; Skipping pod %q", err, podFullName)
```

Comment:
nit: `Failed to setup networking for pod %q using network plugins: %v; Skipping pod`